src/ensembl/src/shared/types/sidebar-toolstrip-collection.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export enum SidebarToolstripCollection {
+  SEARCH = 'search',
+  BOOKMARKS = 'bookmarks',
+  TRACKS_MANAGER = 'tracks-manager',
+  PERSONAL_DATA = 'personal-data',
+  SHARE = 'share',
+  DOWNLOADS = 'downloads'
+}
```

Comment:
Given the `SidebarModalView` enum exported from `entityViewerSidebarState.ts`, what's the purpose of this file?

Reply:
yes, we can remove it.